packages/executable/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  "extends": [
+    "prettier",
+    "standard" 
+  ],
+  "rules": {
+    "no-lone-blocks": 0
```

Comment:
isn't that just a workaround @lebaphi ?

If there was a lone block, we would want `eslint` to tell us about it.

Don't we need to tell eslint that we're using modern JS?